pkgloader-recommended.sty

```diff
@@ -175,6 +175,25 @@
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  A list of recommended rules extracted from the uri
+%  documentation and uses of |\@ifpackageloaded| and |\RequirePackage| in
+%  |uri.sty|:
```

Comment:
Please provide the URL to the documentation. Follow the convention set by the other rules in this file.

(It's http://mirror.ox.ac.uk/sites/ctan.org/macros/latex/contrib/uri/uri.pdf)

Reply:
It's http://texdoc.net/pkg/uri -- http://texdoc.net is the new canonical URL scheme for linking documentation.